std/typecons.d

```diff
@@ -6950,6 +6975,12 @@ template TypedefType(T)
     assert(s2 == cs2);
 }
 
+@safe unittest // toString
```

Comment:
More unit tests are needed that test both overloads of `toString` with multiple different types of values.

Another good test to add would be something like the following to be reasonably sure that the results of `Typedef.toString` are the same as the base type:

```
import std.meta: AliasSeq;
import std.conv;

struct TestS {}
class TestC {}

static foreach (T; AliasSeq!(int, bool, float, double, real, ifloat, idouble, ireal,
                             cfloat, cdouble, creal, char, dchar, wchar, TestS, 
                             TestC, int*, int[], int[2], int[int]))
{{
    T t;
    Typedef!T td
    assert(t.to!string() == td.to!string());
}}
```